proposals/lip-0052.md

```diff
@@ -1478,7 +1496,7 @@ def create(
             "collectionID": collectionID,
             "result": RESULT_SUCCESSFUL
         },
-        topics = [address, nftID]
+        topics = [nftID]
```

Comment:
Why did we remove the address from this event? Sorry if this was from a previous review comment that I missed.

Reply:
My bad, I got confused while reading https://github.com/LiskHQ/lips/pull/179/files/73ac49ad2f4341097be5f723b1dc115d1f98478f#diff-75b2968202aef053f6535bf1d9c00465475cf0ef69c2909f498569b1994227d4. I will return it back to the topics.